client-vms/src/types/types.ts

```diff
@@ -63,7 +63,7 @@ export const PartyName = z.string().min(1);
 export type PartyName = z.infer<typeof PartyName>;
 
 // userid-as-hex/{program-name}/sha256/{sha-of-program}
-export const ProgramId = z.string().min(100);
+export const ProgramId = z.string().min(10);
```

Comment:
This limit was very restrictive. For instance, for the signatures we execute the program 'builtin/ecdsa_sign' and the name has less than 100 chars. In addition, this restriction doesn't exist in the Python client, so I have reduced the minimum number of characters to 10, for now.